core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -1,137 +1,652 @@
 <div class="teach-page">
-  <div class="oppia-static-header">
-    <h1 [innerHTML]="'I18N_TEACH_PAGE_HEADING' | translate"></h1>
+  <div class="oppia-teach-section-one" dir="auto">
+    <div class="oppia-teach-section-one-left">
+      <h1 class="oppia-teach-h1" translate="I18N_TEACH_PAGE_HEADING"></h1>
+      <div class="oppia-teach-h2-container">
+        <h2 class="oppia-teach-h2 oppia-teach-h2-text" translate="I18N_TEACH_PAGE_CONTENT"></h2>
+      </div>
+      <div class="oppia-teach-button-container">
+        <form action="<[$ctrl.classroomUrl]>">
+          <button type="submit" ng-click="$ctrl.onClickStartLearningButton()"
+                  class="btn oppia-teach-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  translate="I18N_TEACH_PAGE_ACTION_START_LEARNING">
+          </button>
+        </form>
+      </div>
+    </div>
+    <div class="oppia-teach-section-one-right">
+      <picture class="oppia-teach-main-image image-desktop">
+        <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/teach/student.jpg')]>">
```

Comment:
No, this should be a webp. You'd need to convert this jpg to a webp copy and store that as well. (you can use the 'mogrify'  cmd line command for that).
Ditto wlsewhere

Reply:
okay.